lib/ansible/modules/cloud/amazon/lambda.py

```diff
@@ -418,7 +456,7 @@ def main():
                     current_version = response['Version']
                 changed = True
             except (botocore.exceptions.ParamValidationError, botocore.exceptions.ClientError) as e:
-                module.fail_json(msg=str(e))
+                fail_json_aws(module, e, msg="updating function code")
```

Comment:
I am not 100% convinced by that.  Firstly I put this in in response to an actual problem (can't remember what right now) so it clearly _can_ happen.  Secondly, AWS can actually change the validation without warning us, so it's possible that what was okay before becomes banned for some good reason.  Do you know some promise from AWS about when they would throw this?